app/components/topbar/TopBar.js

```diff
@@ -1,55 +1,51 @@
 // @flow
 import React, { Component } from 'react';
-import type { MessageDescriptor } from 'react-intl';
 import type { Node } from 'react';
-import { kebabCase } from 'lodash';
 import { observer } from 'mobx-react';
 import TopBarCategory from './TopBarCategory';
 import styles from './TopBar.scss';
-import globalMessages from '../../i18n/global-messages';
 import type { Category } from '../../config/topbarConfig';
-import { GO_BACK_CATEGORIE } from '../../config/topbarConfig';
 
 type Props = {|
   children?: ?Node,
   title: ?Node,
   categories?: Array<Category>,
   activeTopbarCategory: string,
```

Comment:
@leivaburto 
we can remove `activeTopbarCategory`.

Also few waning can also be removed:
```
shin@u-18-04-01:~/repo/emurgo/yoroi-frontend$ npm run eslint

> yoroi@1.8.1 eslint /home/shin/repo/emurgo/yoroi-frontend
> eslint app features chrome scripts webpack/*.js


/home/shin/repo/emurgo/yoroi-frontend/app/components/topbar/TopBar.js
  31:7   warning  'activeTopbarCategory' is assigned a value but never used                                              no-unused-vars
  40:50  warning  Unexpected block statement surrounding arrow body; move the returned value immediately after the `=>`  arrow-body-style

/home/shin/repo/emurgo/yoroi-frontend/app/components/topbar/TopBarCategory.js
  16:3  warning  propType "iconStyle" is not required, but has no corresponding defaultProps declaration  react/require-default-props

/home/shin/repo/emurgo/yoroi-frontend/app/components/widgets/forms/PasswordInstructions.js
  30:13  warning  'intl' is assigned a value but never used  no-unused-vars

/home/shin/repo/emurgo/yoroi-frontend/app/containers/wallet/WalletReceivePage.js
  166:22  warning  A space is required before '}'  object-curly-spacing

/home/shin/repo/emurgo/yoroi-frontend/app/stores/ada/StateFetchStore.js
  15:7  warning  'lastLaunchVersion' is defined but never used. Allowed unused args must match /^_/  no-unused-vars
  16:7  warning  'currentLocale' is defined but never used. Allowed unused args must match /^_/      no-unused-vars

/home/shin/repo/emurgo/yoroi-frontend/app/stores/toplevel/ProfileStore.js
  78:21  warning  Unexpected block statement surrounding arrow body; move the returned value immediately after the `=>`  arrow-body-style

/home/shin/repo/emurgo/yoroi-frontend/app/stores/toplevel/TopbarStore.js
  33:1   warning  Line 33 exceeds the maximum line length of 100                                                         max-len
  41:34  warning  Unexpected block statement surrounding arrow body; move the returned value immediately after the `=>`  arrow-body-style
```